Src/BootCamp.Chapter/FileLogger.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileLogger : ILogger
+    {
+        private readonly string logFile = $"Log_{DateTime.Now.Day}-{DateTime.Now.Month}-{DateTime.Now.Year}.log";
+        private string _message;
+
+        public void SetMessage(string message)
+        {
+            _message = message;
+        }
+
+        public FileLogger()
+        {
+        }
+
+        public FileLogger(string message)
+        {
+            _message = message;
+        }
+
+        public void Log()
+        {
+            using StreamWriter logWriter = new StreamWriter(logFile, true);
```

Comment:
Why StreamWriter and not just File.Write...?

Reply:
I did not use File.Write because it will overwrite the existing file. Using File.AppendAllText rquires to also add a NewLine after each line of text and did not like that. Also I thought about leaving the logFile open (locking the file in the process) and add lines to it without creating new FileLogger objects every time It's needed to write something to log.